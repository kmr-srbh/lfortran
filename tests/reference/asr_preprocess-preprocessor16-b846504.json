{
    "basename": "asr_preprocess-preprocessor16-b846504",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor16.f90",
    "infile_hash": "706f1e5cb790684841189022e27d4df5631d981e6351e1c6fd390bcd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-preprocessor16-b846504.stdout",
    "stdout_hash": "f4671079ac63da27f705b78d60c1236b367a8847e279128b8c4e465a",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}