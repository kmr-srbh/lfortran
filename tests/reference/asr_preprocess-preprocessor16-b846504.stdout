(TranslationUnit
    (SymbolTable
        1
        {
            test_cpp_hex:
                (Program
                    (SymbolTable
                        2
                        {
                            
                        })
                    test_cpp_hex
                    []
                    [(Print
                        [(StringConstant
                            "yes hex and int equal"
                            (Character 1 21 () PointerString)
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "yes hex and bin equal"
                            (Character 1 21 () PointerString)
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "yes bin and int equal"
                            (Character 1 21 () PointerString)
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
